Serialization.NET/Segment/Serialization/JsonUtility.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Text;
 using System.Text.Json;
+using System.Xml;
```

Comment:
is this unused import?

Reply:
Yeah, accidentally pulled in when I used the wrong 'indent' parameter.  Removed.